### ISSUE TITLE
Navs have aria-label="Global"

### ISSUE THREAD

**Issue by quarckster** [patternfly/patternfly-react#1386]
Browser tests failed on navs page due to changed locators. Now all navs have `aria-label="Global"`. It would be nice if they have old values.
https://travis-ci.org/quarckster/widgetastic.patternfly4/jobs/493297943#L1433

**Comment by rachael-phillips** [comment]
Hi @quarckster this was intentionally changed in https://github.com/patternfly/patternfly-next/issues/1127 and #1182 . I'm going to go ahead and close this issue. Please feel free to reopen if you have any more questions. Thanks!
